Demo/Sources/ContentLists/ContentListView.swift

```diff
@@ -58,12 +72,17 @@ private struct ContentCell: View {
     var body: some View {
         switch content {
         case let .topic(topic):
+#if os(iOS)
             NavigationLink(
                 topic.title,
                 destination: .contentList(configuration: .init(list: .latestMediasForTopic(topic), vendor: topic.vendor))
             )
-#if os(iOS)
             .swipeActions { CopyButton(text: topic.urn) }
+#else
+            NavigationLink(destination: RouterDestination.contentList(configuration: .init(list: .latestMediasForTopic(topic), vendor: topic.vendor)).view()) {
```

Comment:
```suggestion
            NavigationLink(destination: .contentList(configuration: .init(list: .latestMediasForTopic(topic), vendor: topic.vendor)).view()) {
```

Reply:
Unfortunately, in this case, the type cannot be inferred because the `NavigationLink` is waiting for a `View`.